src/pybind/mgr/nfs/module.py

```diff
@@ -137,9 +137,11 @@ def _cmd_nfs_cluster_info(self, cluster_id: Optional[str] = None) -> Dict[str, A
         return self.nfs.show_nfs_cluster_info(cluster_id=cluster_id)
 
     @CLICommand('nfs cluster config get', perm='r')
+    @object_format.ErrorResponseHandler()
```

Comment:
I don't really understand what's different about `_cmd_nfs_cluster_config_get` that means it needs this decorator rather than the normal `Responder`. There are other function that call functions that can raise `ErrorResponse.wrap(e)` that just use the normal `Responder` (e.g. `_cmd_nfs_cluster_ls`).

Reply:
So the end-user cli behavior is for `ceph nfs cluster config get` to dump raw ganesha config data to stdout. Responder wants to return some sort of structured data (json/yaml/etc). So simply using responder would give us the ganeshs config wrapped in a JSON/YAML strcuture which would be a behavior change.  We could try to change it so that we use Responder with a plain-text format. I think that would work OK but it seemed like more work would be needed to implement that and this was a smaller change in the code.  I figured a smaller delta would mean fewer chances for regressions here.

Reply:
To follow up: the Responder would *require* plain text, or at the very least need to default to it. And that would be net-new code for object_format too.

Reply:
ah, okay. I wasn't aware this command returned structured output already. In that case, I see why we wouldn't want to use the normal Responder.